apps/docs/src/app/platform/component-docs/platform-split-menu-button/platform-split-menu-button-examples/platform-split-menu-button-examples.component.html

```diff
@@ -1,98 +1,197 @@
+</div>
+<br />
+
+<div>
+    <fdp-split-menu-button id="splitButton9" [menu]="basicMenuCompactNegative" [buttonLabel]="labelNegativeCompact"
+        icon="world" type="negative" title="Default action button with list of other available action items"
+        menuTitle="More options" contentDensity="compact" (primaryButtonClick)="onPrimaryButtonClickCompactNegative()">
+    </fdp-split-menu-button>
+</div>
+<br />
+
+<div>
+    <fdp-split-menu-button id="splitButton10" [menu]="basicMenuCompactGhost" [buttonLabel]="labelGhostCompact"
+        icon="world" type="ghost" contentDensity="compact"
+        title="Default action button with list of other available action items" menuTitle="More options"
+        (primaryButtonClick)="onPrimaryButtonClickCompactGhost()">
+    </fdp-split-menu-button>
+</div>
+<br />
+
+<div>
+    <fdp-split-menu-button id="splitButton11" [menu]="basicMenuCompactTransparent"
+        [buttonLabel]="labelTransparentCompact" icon="world" contentDensity="compact" type="transparent"
+        title="Default action button with list of other available action items" menuTitle="More options"
+        (primaryButtonClick)="onPrimaryButtonClickCompactTransparent()">
+    </fdp-split-menu-button>
+</div>
+<br />
+
+<div>
+    <fdp-split-menu-button id="splitButton12" [menu]="basicMenuCompactEmphasized" [buttonLabel]="labelEmphasizedCompact"
+        icon="world" contentDensity="compact" type="emphasized"
+        title="Default action button with list of other available action items" menuTitle="More options"
+        (primaryButtonClick)="onPrimaryButtonClickCompactEmphasized()">
     </fdp-split-menu-button>
+</div>
 
-    <span>&nbsp;&nbsp; {{ buttonType | titlecase }}</span>
-    <br />
-    <br />
-</ng-container>
-
-<fdp-split-menu-button
-    [id]="'split-menu-button3'"
-    [name]="'split-menu-button3'"
-    [menu]="basicMenuCompact"
-    [contentDensity]="'compact'"
-    [disabled]="true"
-    [buttonLabel]="label"
-    [icon]="'world'"
-    (primaryButtonClick)="onPrimaryButtonClickCompact()"
->
+<br />
+<fdp-split-menu-button id="splitButton13" contentDensity="compact" [disabled]="true" [buttonLabel]="labelStandardCozy"
+    icon="world" contentDensity="compact">
 </fdp-split-menu-button>
-<span>&nbsp;&nbsp;Disabled</span>
 <br />
 <br />
 <p>Selected Item: {{ selectedCompactItem }}</p>
 
-<h3>Internationalisation support with i18n directive</h3>
-<fdp-split-menu-button
-    [id]="'split-menu-button4'"
-    [name]="'split-menu-button4'"
-    [menu]="basicMenuCompact"
-    [icon]="'world'"
-    i18n-buttonLabel
-    buttonLabel="Default Action"
-    i18n-title
-    title="Default action button with list of other available action items"
-    (primaryButtonClick)="onPrimaryButtonClickCompact()"
-    i18n="@@basicSplitMenuBtn"
->
+<h3>Internationalization support with i18n directive</h3>
+<fdp-split-menu-button id="splitButton14" [menu]="basicMenuIntz" icon="world" i18n-buttonLabel [buttonLabel]="label"
```

Comment:
Do we really need to assign all these `ids`, as for me it bloats examples and doesn't bring any value.

Reply:
removed ids.